Tests/KernelTestCaseTrait.php

```diff
@@ -0,0 +1,313 @@
+    }
+
+    public static function terminateKernel()
+    {
+        if (self::$kernel) {
+            self::$kernel->free();
+            self::$kernel = null;
+            gc_collect_cycles();
+        }
+    }
+
+    /**
+     * @param string $environment
+     * @return Kernel
+     */
+    public static function getKernel($environment = 'swagconnecttest')
+    {
+        if (!self::$kernel) {
+            self::bootKernel($environment);
+        }
+
+        return self::$kernel;
+    }
+
+    /**
+     * @param string $filePath
+     */
+    protected function importFixturesFileOnce($filePath)
+    {
+        if (!file_exists($filePath)) {
+            throw new \InvalidArgumentException('Could not find file ' . $filePath);
+        }
+
+        if (array_key_exists($filePath, $this->importedFiles)) {
+            return;
+        }
+
+        $this->importedFiles[$filePath] = true;
+
+        if (!$this->inTransaction) {
+            return;
+        }
+
+        $sql = file_get_contents($filePath);
+
+        if (!$sql) {
+            return;
+        }
+
+        $this->importFixtures($sql);
+    }
+
+    /**
+     * @return bool
+     */
+    public function isShopware53()
+    {
+        return self::getKernel()->getContainer()->has('customer_search.dbal.number_search');
+    }
+
+    /**
+     * @param string $sql
+     * @throws \Exception
+     */
+    public function importFixtures($sql)
+    {
+        $defaultSql = $this->loadCommonFixtureSql();
+
+        if ($defaultSql) {
+            $sql = $defaultSql . $sql;
+        }
+
+        $connection = self::getKernel()->getContainer()->get('dbal_connection');
+
+        $sqlStatements = explode(";\n", $sql);
+
+        foreach ($sqlStatements as $sqlStatement) {
+            if (!trim($sqlStatement)) {
+                continue;
+            }
+
+            $connection->exec(trim($sqlStatement));
+        }
+
+        if (! (int) $connection->errorCode()) {
+            return;
+        }
+
+        throw new \Exception('unable to import fixtures ' . print_r($connection->errorInfo(), true));
+    }
+
+    public function performApiLogin()
+    {
+        \Zend_Session::$_unitTestEnabled = true;
+
+        $adapter = new class implements \Zend_Auth_Adapter_Interface {
+            /**
+             * Performs an authentication attempt
+             *
+             * @throws \Zend_Auth_Adapter_Exception If authentication cannot be performed
+             * @return \Zend_Auth_Result
+             */
+            public function authenticate()
+            {
+                return new \Zend_Auth_Result(\Zend_Auth_Result::SUCCESS, ['id' => 1, 'username' => 'demo']);
+            }
+        };
+
+        $auth = \Shopware_Components_Auth::getInstance();
+        $auth->setBaseAdapter($adapter);
+        $auth->addAdapter($adapter);
+
+        self::getKernel()->getContainer()->set('auth', $auth);
+    }
+
+    public function performB2bContactLogin()
```

Comment:
i think we don't need these b2b specific methods 
please remove evrything that we don't need